upup/pkg/fi/context.go

```diff
@@ -158,7 +158,7 @@ func (c *Context) Render(a, e, changes Task) error {
 				b.WriteTo(out)
 
 				if *lifecycle == LifecycleExistsAndValidates {
-					return fmt.Errorf("Lifecycle set to ExistsAndValidates, but object did not match")
+					return fmt.Errorf("lifecycle set to ExistsAndValidates, but object did not match")
```

Comment:
Can you tell the reason of change the word to lowercase, please?

Reply:
This is a static inspection specification.

Reply:
As you can see, it is just a specification.